ui/espresso/IntentsAdvancedSample/app/src/androidTest/java/com/example/android/testing/espresso/intents/AdvancedSample/ImageViewHasDrawableMatcher.java

```diff
@@ -16,6 +16,7 @@
 
 package com.example.android.testing.espresso.intents.AdvancedSample;
 
+import androidx.test.espresso.matcher.BoundedDiagnosingMatcher;
 import androidx.test.espresso.matcher.BoundedMatcher;
```

Comment:
Removes the BoundedMatcher import as it's no longer needed?

Reply:
Done!